ingestion/src/metadata/data_quality/validations/base_test_handler.py

```diff
@@ -85,12 +85,15 @@ def get_test_case_param_value(
         pre_processed_value = pre_processor(value)
         return type_(pre_processed_value)
 
-    def get_test_case_result_object(
+    def get_test_case_result_object(  # pylint: disable=too-many-arguments
         self,
         execution_date: Union[datetime, float],
         status: TestCaseStatus,
         result: str,
         test_result_value: List[TestResultValue],
+        row_count: Optional[int] = None,
+        failed_rows: Optional[int] = None,
+        passed_rows: Optional[int] = None,
```

Comment:
nit - should this be a custom model? RowsResult or something, this way we can keep less args

Reply:
I'll do it in another PR to not block the UI